website-1.x/blog/2020-01-12-docusaurus-2020-recap.md

```diff
@@ -0,0 +1,82 @@
+
+We are **still in alpha**, but expect **some good news very soon**!
+
+<!--truncate-->
+
+## Docusaurus 2 highlights
+
+We have worked on many features this year, and would like to highlight the most significant ones:
+
+- **i18n**: easily translate your Docusaurus site, released soon! ([preview](https://github.com/facebook/docusaurus/pull/3325))
+- **Version dropdown**: enable contextual version switch
+- **Versioning options**: solving the most common pain-points users had with versioning
+- **Contextual search**: search in current version, use the brand-new Algolia DocSearch 3 experience
+- **Markdown pages**: use Markdown instead of React components to create standalone pages
+- **PWA**: add offline support to your site, and make it installable
+- **Fail-fast**: strict config validation, detect broken links and prevent bad production deployments
+- **Multi-instance**: use the docs plugin twice on a single Docusarus site (e.g. ios/android doc sdks)
+- **Migration CLI**: automate many parts to upgrade easily from Docusaurus 1
+- **CodeSandbox**: assess Docusaurus easily from your browser
+- **Canary releases**: use the `@canary` npm tag to use the upcoming release before everyone else
+
+## Docusaurus 2 growth
+
+The plan to [rebuild Docusaurus from scratch in 2019](https://docusaurus.io/blog/2019/12/30/docusaurus-2019-recap) paid off: after a slow start, Docusaurus 2 has been widely adopted and has **already outgrown Docusaurus 1** usage.
+
+![Docusaurus v1 vs v2 npm trends](/img/blog/2020-recap/docusaurus-npm-trends.png)
+
+Notable **projects, startups, large companies and individuals** adopted Docusaurus 2 ([showcase](https://v2.docusaurus.io/showcase)):
+
+- [Supabase](https://supabase.io)
+- [React-Navigation](https://reactnavigation.org)
+- [React-Redux](https://react-redux.js.org/)
+- [Vector](https://vector.dev)
+- [Algolia DocSearch](https://docsearch.algolia.com)
+- [SAP Cloud SDK](https://sap.github.io/cloud-sdk)
+- [Palo Alto Cortex XSOAR](https://xsoar.pan.dev)
+- [Quddús George's personal website](https://portfoliosaurus.now.sh) s We helped **large scale Docusaurus 1 sites to upgrade**, ensuring a proper **migration path** and **feature parity**.
+
+[ReactNative](https://reactnative.dev/) was successfully migrated ([old version](http://archive.reactnative.dev/)).
+
+![ReactNative screenshot](/img/blog/2020-recap/react-native-screenshot.png)
+
+Localized Docusaurus 1 sites like **Jest** will be able to upgrade too:
+
+![Jest in Japanese screenshot](/img/blog/2020-recap/jest-screenshot.png)
+
+We even saw the **[first right-to-left](https://datagit.ir/)** Docusaurus 2 site published:
+
+![Datagit screenshot](/img/blog/2020-recap/datagit-rtl-screenshot.png)
+
+## GitHub Activity
+
+- **Stars**: 14632 -> 20946 (+43.2% y/y)
+- **Total Contributors**: 303 -> 512 (+68.9% y/y). Most of which are non-Facebook contributors
+- **Daily npm Downloads**: ??? TODO
```

Comment:
- **Weekly npm Downloads**: 2356 -> 25592 (+TODO y/y)

Reply:
thanks.
Wonder what tools you use to get these data?

Reply:
Just the npm package page and the repo insights tab, nothing special. I wanted to get more data via our internal OSS repo dashboards but they're down at the moment. I guess these numbers are sufficient anyway :D